lib/msf/core/modules/external/python/metasploit/module.py

```diff
@@ -14,7 +14,7 @@ def __init__(self, prefix, *args, **kwargs):
         self.prefix = prefix
 
     def format(self, record):
-        return self.prefix + record.msg
+        return self.prefix + (record.msg % record.args)
```

Comment:
`logging.Formatter` has a number of styles it can be initialized with, so this should be a `super` call:
```suggestion
        return self.prefix + super().format(record)
```